.github/workflows/ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ v1-mvp ]
+
+jobs:
```

Comment:
What individual jobs will be running every time? Each job is a separate task to be done.